spec/views/sign_up/select_email/show.html.erb_spec.rb

```diff
@@ -10,11 +11,16 @@
     create(:email_address, email: email2, user:)
     @user_emails = user.confirmed_email_addresses
     @select_email_form = SelectEmailForm.new(user:)
+    @sp_name = 'Test Service Provider'
   end
 
-  it 'shows all of the user\'s emails' do
-    render
+  it 'renders introduction text' do
+    expect(rendered).to have_content(
+      strip_tags(t('help_text.select_preferred_email_html', sp: @sp_name)),
+    )
+  end
 
+  it 'shows all of the user\'s emails' do
```

Comment:
Personally I find the quote escaping distracting, maybe we could rephrase so we don't need to use any quotes
```suggestion
  it 'shows all of the emails' do
```

Reply:
> Personally I find the quote escaping distracting, maybe we could rephrase so we don't need to use any quotes

Sure, I agree on it being a distraction 👍 Updated in a5f535c.